migration/init.go

```diff
@@ -0,0 +1,22 @@
+package migration
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/gconf"
+)
+
+// Initializer fulfils the InitStater interface to load data from
+// the genesis file
+type Initializer struct{}
+
+var _ weave.Initializer = Initializer{}
+
+// FromGenesis will parse initial account info from genesis
+// and save it to the database
+func (Initializer) FromGenesis(opts weave.Options, kv weave.KVStore) error {
+	if err := gconf.InitConfig(kv, opts, "migration", &Configuration{}); err != nil {
```

Comment:
Just realized this was not tested yet...

We will have to include the schema info in the genesis as well???
Or we have some trick here?
Or we don't version the config? (actually would be great to do so if possible)

Reply:
Yes, schema registration must be somehow included. I am going to work on this part today. This is also mandatory in order for the `bcpd` to be able to work.